src/mavsdk/plugins/camera/camera_impl.cpp

```diff
@@ -506,11 +506,6 @@ Camera::Result CameraImpl::stop_video()
 {
     auto cmd_stop_video = make_command_stop_video();
 
-    {
```

Comment:
stop video just mean stop recording video to file, so not need stop video streaming.